stories/button/button.stories.js

```diff
@@ -284,7 +284,7 @@ menuButton.parameters = {
 
 export const splitMenuButton = () => `
 <div class="fd-button-split fd-has-margin-right-small" role="group" aria-label="button-split">
-  <button class="fd-button" aria-label="button">Button with text</button>
+  <button class="fd-button fd-button-split__text" aria-label="button">Button with a lot of text</button>
```

Comment:
Could you just wrap the text inside `<span class="fd-button__text">Button with a lot of text</span>`?
And inside split-button just add `fd-button__text { max-width: ... }`

Reply:
fixed